wolox-books/src/scss/_fonts.scss

```diff
@@ -0,0 +1,2 @@
+$ContentText: 16px;
```

Comment:
variables shouldn't be in camelCase, by the way, name it in lowercase, don't use uppercase in variable names